pkg/cli/mirror/prune.go

```diff
@@ -181,10 +184,5 @@ func (p *manifestDeleter) DeleteManifest(repo, manifest string) error {
 		return fmt.Errorf("error parsing image reference %s: %v", ref, err)
 	}
 
-	err = remote.Delete(nameRef, p.ropts...)
-	if errors.As(err, &terr) {
-		fmt.Fprintf(p.w, "WARNING: Pruning failed for image %q with %d response code\n", ref, terr.StatusCode)
```

Comment:
There was some discussion about adding additional context to this warning. With some directions to the user to either perform this manually or increase permissions. Was there something else decided after our conversation this morning?

Reply:
Based on the conversation yesterday, I think some wording to direct the user to manually remove the image, and change permissions would be helpful. But, if the error message gives enough information to help the us understand what the issue is, then I am good with this update, and you can mark this resolved.